Fitness/Extensions/Date+Shared.swift

```diff
@@ -44,6 +44,28 @@ extension Date {
         
         return dateFormatter.date(from: datetime) ?? Date()
     }
+    
+    static public func getTimeFromString(datetime: String?) -> Date {
+        guard let datetime = datetime else {
+            return Date()
+        }
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "HH:mm:ss"
+        
+        let date = dateFormatter.date(from: datetime) ?? Date()
+        let today = Date()
+        let calendar = Calendar.current
+        var dateComponents = DateComponents()
+        
+        dateComponents.year = calendar.component(.year, from: today)
+        dateComponents.month = calendar.component(.month, from: today)
+        dateComponents.day = calendar.component(.day, from: today)
+        dateComponents.timeZone = TimeZone(abbreviation: "EDT")
```

Comment:
what happens if I am in the west coast but open the app? Will it just show all the times wrong?